.github/scripts/run_ft_tests.bash

```diff
@@ -20,5 +20,4 @@ killandsleep () {
 }
 
 cd ${NODE_PATH}/@zenko/cloudserver && yarn run mem_backend & bash tests/utils/wait_for_local_port.bash $PORT 40
-./node_modules/.bin/nyc --clean --silent yarn run $1
-./node_modules/.bin/nyc report --report-dir "./coverage/$1" --reporter=lcov
+./node_modules/.bin/c8 --report-dir "./coverage/$1" --reporter lcov --clean yarn run $1
```

Comment:
why did you change this?

Reply:
Just testing things out to see if maybe there was an issue in merging the coverage of c8 and nyc